### ISSUE TITLE
Adding relevant labels (low priority but still..)

### ISSUE THREAD

**Issue by nogainbar** [InFact-coop/InFact#16]
Might be very low priority but i am not a github labels expert. how do we copy paste relevant labels from FAC and DWYL organization repo's so that we can label our issues?

**Comment by bradreeder** [comment]
@nogainbar dwyl have built there own tool for copying their labels:

https://github.com/dwyl/labels

**Comment by nogainbar** [comment]
@bradreeder I know but we probably want different labels for this repo that are more to do with organising rather than writing code.. ?

**Comment by jsms90** [comment]
@nogainbar agreed. That's why I deleted all the default github label from this repo - they're too code specific. Labels like "question" & "help wanted" might be useful in a coding context, to differentiate between issues. In a business context, everything is a question/discussion.

[Our labels in FAC](https://github.com/foundersandcoders/london-programme/labels) aren't that useful imo (happy to rectify that too. Just not sure how. We just have 114 open issues as it is :sweat_smile:) - very similar colours for very different things, lots that we don't use, etc.

[DWYL's labels](https://github.com/dwyl/start-here/labels) are for development. As far as I know, they don't make business decisions via github. And if they do, they must not use those labels.

I suggest that we create our own, rather than porting in FAC or DWYL ones.

**Comment by jsms90** [comment]
So far, I created one: **community**. This is essentially "networking"...but that word has all the wrong connotations. No one likes networking i.e. schmoozing. Whereas "community" is about us reaching out to the wider co-operative community, or tech community as friends. It's green because we're growing something here & green=nature :blush: 

I'm clearly getting very hippy with this :stuck_out_tongue_closed_eyes:

**Comment by bradreeder** [comment]
@jsms90 @nogainbar 

I'm probably going to go live in a forest commune for a week in May before / after joining @nogainbar in the woods for a few days. The hippyness is basically inevitable.

**Comment by nogainbar** [comment]
I think there are some labels we can take, like the 'priority 1,2,3..' and also 'question'.. agree we can make our owns and that could be good learning too.

**Comment by nogainbar** [comment]
@bradreeder thats the spirit. 😂

**Comment by jsms90** [comment]
Dark purple? :flushed:

**Comment by jsms90** [comment]
What should our `project` label be? For #22 and #21

**Comment by nogainbar** [comment]
added some labels!! 👻

**Comment by nogainbar** [comment]
im high on the purples and blues! :)
